packages/expo-auth-session/src/TokenRequest.types.ts

```diff
@@ -74,6 +74,12 @@ export interface AccessTokenRequestConfig extends TokenRequestConfig {
    * [Section 3.1.2](https://tools.ietf.org/html/rfc6749#section-3.1.2)
    */
   redirectUri: string;
+  /**
+   * If we are using PKCE then we include the original 'code_verifier' in the token request.
```

Comment:
If possible, could you add a more informative doc block -- for instance I think people may not know what the original `code_verifer` is if they use the hooks API.